README.md

```diff
@@ -1,20 +1,53 @@
 # elrrdatasync
 ELRR services which aid in the streaming (Kafka streams) of data from the staging database to the ELRR database.
 
-# Requirements
-For building and running the elrrdatasync you need:
-- JDK 1.8
-- Maven 3
+There are database and kafka dependencies, but there's a repo with a docker-compose that resolves them locally.
+[.yaml](https://github.com/US-ELRR/elrrdatasync/tree/dev-pom-update-local-fixes-and-docs-1/dev-resources/docker-compose)
```

Comment:
#1: this is a super strange way to make a link visible:
![Screenshot from 2024-10-29 11-03-45](https://github.com/user-attachments/assets/62a1763a-bf38-47cf-90e7-f3fc3d764a53)

Just link it inline like "...there's a repo with a [docker-compose.yml] that resolves..."

#2 you used branch-specific links here. That's not good going forward because they will ALWAYS link back to a dead branch from master/continuing docs. Same thing with the datasync link below. In both cases simply link to the root of the repo, as the resource will be evident but always up to date.